components/fieldlabel/index.css

```
@@ -10,47 +10,83 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-@import "../vars/css/components/spectrum-fieldlabel.css";
+
+/* Component level tokens currently missing in core-tokens as of July 2022 */
+.spectrum--medium {
+  --spectrum-field-label-top-to-asterisk-small: 8px;
+}
+.spectrum--large {
+  --spectrum-field-label-top-to-asterisk-small: 11px;
+}
+
 
 .spectrum-FieldLabel--sizeS {
-  @remapvars {
-    find: --spectrum-fieldlabel-s-;
-    replace: --spectrum-fieldlabel-;
-  }
+  --spectrum-fieldlabel-top-to-text: var(--spectrum-component-top-to-text-75);
+  --spectrum-fieldlabel-bottom-to-text: var(--spectrum-component-bottom-to-text-75);
+  --spectrum-fieldlabel-font-size: var(--spectrum-font-size-75);
+
+  --spectrum-fieldlabel-line-height: var(--spectrum-line-height-100);
+  --spectrum-fieldlabel-CJK-line-height: var(--spectrum-CJK-line-height-100);
```

I'm trying to catch as many of these as I can, and I know that the core tokens value _is_ capitalized, but I think we should standardize on making it lowercase when we create our own aliases.

```suggestion
  --spectrum-fieldlabel-cjk-line-height: var(--spectrum-CJK-line-height-100);
```

---

made it as lower case as possible...